### DISCUSSION TITLE
Usage mlflow with lakeFS

### DISCUSSION THREAD

**Discussion by PeterPirog** [mlflow/mlflow#6622] [Ideas]
Are there any plans to integrate mlflow with lakeFS [https://github.com/treeverse/lakeFS](https://github.com/treeverse/lakeFS) ?

LakeFS can be interesting tool to improve mlflow databases  and save disk space  for large space datasets.

**Comment by JHibbard** [comment]
Hi PeterPirog, this is an interesting idea. I think creating a SQLAlchemy dialect for lakeFS and other file storage formats would be a good approach to this. There's currently no plans to implement this that I know of, but it would be a great OSS contribution.
